pages/gigabit/About.jsx

```diff
@@ -44,7 +44,7 @@ module.exports = React.createClass({
         </div>
 
         <div className="clearfix text-center">
-          <img className="w-75 p-a-2" src="/img/pages/gigabit/map-v2.svg"/>
+          <img className="img-responsive m-x-auto p-a-2" src="/img/pages/gigabit/map-v2.svg"/>
```

Comment:
I wasn't sure what the w-75 was for. `img-responsive` gives max-width so the image scales down on mobile.